common/persistence/dataInterfaces.go

```diff
@@ -2538,3 +2549,13 @@ func NewGetReplicationTasksFromDLQRequest(
 		},
 	}
 }
+
+type ServiceType int
+
+const (
+	Unknown ServiceType = iota
```

Comment:
Maybe call Unknown as All?  As the behavior is to return all hosts irrespective of the role.

Reply:
Moved to All, however I think we may want to have a default of None/Unknown = defaultValue for any type that we persist (may just want to do it everywhere). This is a practice is some circles but lets discuss.